controllers/hibp.js

```diff
@@ -37,6 +37,13 @@ async function notify (req, res) {
     }
   }
 
+  if (breachAlert.IsSpamList || breachAlert.Domain === "" || breachAlert.IsFabricated) {
```

Comment:
Q: Should we move this logic into some helper?
It looks like we're using it here, and a similar (but inverted) version below in /hibp.js:L104-107.

Reply:
Yes, that would certainly be cleaner. Did a bit of DRYing up in HIBP in 24450c4 with this in mind but it doesn't help us out any here.